lib/utils/schematic/getAllDimensionsForSchematicBox.ts

```diff
@@ -194,8 +194,13 @@ export const getAllDimensionsForSchematicBox = (
       truePinIndex,
     })
 
+    const pinLabel =
+      params.pinLabels?.[`pin${pinNumber}`] ?? params.pinLabels?.[pinNumber]
```

Comment:
this doesn't handle the case where `pinLabels: Record<string, string[]>`

Reply:
adding a case for that 👍🏻

Reply:
added a case handle their